.travis.yml

```diff
@@ -1,12 +1,22 @@
+sudo: false
+
 language: node_js
 
 node_js:
   - "0.10"
 
+addons:
+  apt:
+    sources:
+      - git-core
```

Comment:
`git-core` is the only one [in the whitelist](https://github.com/travis-ci/apt-source-whitelist/blob/dce2f6b8fd2ccaf4fcd41dbe2f17c69e80d6d6a9/ubuntu.json#L123), so maybe it's true for regular apt, but not with the docker packages.